packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -318,88 +485,37 @@ const testUpgrade = async (t, defaultManagerType, options = {}) => {
-     long as non-durable object cleanup in stop-vat is also disabled.
-
-  // of the durables, only these survive
-  const survivingDurables = [
-    1, 16, 17, 18, 19, 20, 26, 27, 28, 33, 34, 35, 36, 37,
-  ];
-  // and these imports (imp38 is held by baggage)
-  const survivingImported = [
-    1, 16, 17, 18, 19, 20, 26, 27, 28, 33, 34, 35, 36, 37, 38,
-  ];
-
-  // but implementation limitations/bugs cause the following unwanted
-  // effects (these adjustments should be deleted as we fix them):
-
-  // stopVat() uses deleteVirtualObjectsWithoutDecref, rather than
-  // deleteVirtualObjectsWithDecref, which means lingering virtual
-  // objects (i.e. cycles) don't drop their referenced objects as we
-  // delete them
-  survivingDurables.push(9);
-  survivingImported.push(7);
-  survivingImported.push(8);
-  survivingImported.push(9);
-
-  // When a virtual collection is deleted, the loop that deletes all
-  // entries will re-instantiate all the keys, but doesn't set
-  // doMoreGC, so processDeadSet doesn't redo the gcAndFinalize, and
-  // the virtual object cache is probably still holding onto the new
-  // Representative anyways. This retains the durables that were held
-  // by deleted collections (dur10/dur13/dur23, depending on the cache
-  // size, just dur23 if size=0) and the imports they hold. Bug #5053
-  // is about fixing clearInternal to avoid this, when that's fixed
-  // these should be removed.
-  survivingDurables.push(10);
-  survivingImported.push(10);
-  survivingDurables.push(13);
-  survivingImported.push(13);
-  survivingDurables.push(23);
-  survivingImported.push(23);
-
-  for (let i = 1; i < NUM_SENSORS + 1; i += 1) {
-    const vref = durVref(i);
-    // const impKref = impKrefs[i];
-    const expD = survivingDurables.includes(i);
-    // const expI = survivingImported.includes(i);
-    // const reachable = krefReachable(impKref);
-    t.is(vomHas(vref), expD, `dur[${i}] not ${expD}`);
-    // t.is(reachable, expI, `imp[${i}] not ${expI}`);
-    // const abb = (b) => b.toString().slice(0,1).toUpperCase();
-    // const vomS = `vom: ${abb(expD)} ${abb(vomHas(vref))}`;
-    // const reachS = `${abb(expI)} ${abb(reachable)}`;
-    // const match = (expD === vomHas(vref)) && (expI === reachable);
-    // const matchS = `${match ? 'true' : 'FALSE'}`;
-    // const s = kvStore.get(`${vatID}.c.${impKref}`);
-    // console.log(`${i}: ${vomS} imp: ${reachS} ${matchS}  ${impKref} ${s}`);
-  }
-  */
-
-  // check koNN.owner to confirm the exported virtuals (2/5/7) are abandoned
-  t.false(kvStore.has(`${vir2Kref}.owner`));
-  t.false(kvStore.has(`${vir5Kref}.owner`));
-  t.false(kvStore.has(`${vir7Kref}.owner`));
+  // verify export abandonment/garbage collection/etc.
+  // This used to be MUCH more extensive, but GC was cut to the bone
+  // in commits like 91480dee8e48ae26c39c420febf73b93deba6ea5
+  // basically reverting 1cfbeaa3c925d0f8502edfb313ecb12a1cab5eac
+  // (see also #5342 and #6650).
```

Comment:
Thanks for adding those notes. It'd also be nice to have a pointer to the git commit-ID from shortly before this lands, so exclude the various test changes that have happened since those two commits. Either cite this PR number (from which we can figure it out via github history), or sample current trunk's commit-ID (`ec6a7f4eded916afb8c927638b4a699e2ffd99c9` as of this morning, which is close enough).